docs/jwst/user_documentation/jwst_helpdesk.rst

```diff
@@ -0,0 +1,7 @@
+==============
+JWST HELP DESK
+==============
+
+If you have questions or concerns regarding the software, please open an issue
+at https://github.com/spacetelescope/jwst/issues or contact
```

Comment:
Actually, I don't really want to encourage generic users all over the world to create github issues. It's better for our internal tracking if they go through the help desk and then we create a JP ticket when appropriate. So I'd change this to simply read "If you have questions or concerns regarding the software, please contact the `JWST Help Desk`."

Reply:
@nden  @hbushouse I took that statement from the user_documentation/available_pipelines.rst 
Should I also change the wording in that document ?

Reply:
I'm fine with mentioning github down there if @hbushouse agrees. For some people I think it's preferable to write github issues